lang/rs/apis/cne/src/instance.rs

```diff
@@ -54,6 +57,14 @@ impl CneInstance {
 
 // CneInstance public functions.
 impl CneInstance {
+    /// Configures CNE instance using JSONC configuration file.
+    ///
+    /// # Arguments
+    /// * `jsonc_file` - jsonc configuration file path.
+    ///
+    /// # Errors
+    /// Returns [CneError::ConfigError] if an error is encountered.
```

Comment:
Where does the CneError::ConfigError come from all I can tell is OK is being returned. Most likely I am not understanding something about Rust.

Reply:
The question mark operator `?` is used in this function to propagate errors. [Rust question mark operator](https://doc.rust-lang.org/book/ch09-02-recoverable-errors-with-result.html#a-shortcut-for-propagating-errors-the--operator)